src/kivy_garden/zbarcam/zbarcam.py

```diff
@@ -210,3 +211,19 @@ def stop(self):
         self.xcamera.play = False
         if platform == "android":
             self.xcamera._camera._release_camera()
+
+    def unload_zbarcam(self):
+        id ZBarCam.kv_loaded:
```

Comment:
Did you mean `if`?
Pity we no longer have working tests for this repo, this would have been caught easily

Reply:
Yes, sorry for typo.

I will fix this and create a new commit

Reply:
> Pity we no longer have working tests for this repo, this would have been caught easily

Maybe I can work with some github-action to do this.

Reply:
Yeah we would love to :+1: